ebpf.mk

```diff
@@ -11,7 +11,9 @@ build:
 	@sudo mkdir -p "$(DEST_DIR)"
 	clang -D__KERNEL__ -D__ASM_SYSREG_H -D__BPF_TRACING__ \
 		-DCIRCLE_BUILD_URL=\"$(CIRCLE_BUILD_URL)\" \
+		-DDEBUG=1 \
```

Comment:
Won't this cause the library we are shipping to also be compiled with DEBUG symbols.